### ISSUE TITLE
is GoLogin paying attention to this repo?

### ISSUE THREAD

**Issue by alvinlys** [gologinapp/gologin#126]
Hi, it seems there are many issue raised but none from the author or maintainer is helping it.

**Comment by iamkhalidbashir** [comment]
I think not

**Comment by alvinlys** [comment]
Got this reply from their support team, i guess its time to refund back my subscription
<img width="374" alt="image" src="https://github.com/gologinapp/gologin/assets/32334860/7c4a1318-2d64-4161-b21c-46d1428032f3">

**Comment by igortikushEasync** [comment]
Yes - we paying attention to this repo. But it's not our top priority.
